src/feedlistformaction.cpp

```diff
@@ -273,7 +273,8 @@ bool FeedListFormAction::process_operation(Operation op,
 					v->show_error(_("Failed to spawn browser"));
 					return false;
 				} else if (*exit_code != 0) {
```

Comment:
Nitpickky suggestion for consistency:
```suggestion
				} else if (exit_code.value() != 0) {
```

Reply:
Yeah, makes sense, thank you!